### PR TITLE
add python3-jsonschema to  dockers/docker-platform-monitor/Dockerfile.j2

### PR DESCRIPTION
**Pull request by VenkatCisco**
#### Why I did it
jsonschema is an implementation of JSON Schema for Python .

#### How I did it
added python-jsonschema to docker-platform-monitor/Dockerfile.j2 

#### How to verify it

#!/usr/bin/env python

from jsonschema import validate

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [x] 202006
- [x] 202012

#### Description for the changelog

jsonschema is an implementation of JSON Schema for Python .

```
root@sonic:/# du -ch / | grep python-jsonschema
8.0K    /usr/share/doc/python-jsonschema
```

### CODE REVIEW

**Comment by jleveque** [comment]
@VenkatCisco: In the PR description, can you please indicate the installed size of this package on disk?

**Comment by VenkatCisco** [comment]
> @VenkatCisco: In the PR description, can you please indicate the installed size of this package on disk?

Done.

**Review by jleveque — DISMISSED** [review]

**Comment by jleveque** [comment]
@VenkatCisco: Is it necessary to backport this to the 201911 branch? That branch is relatively stable; we only want to backport bug fixes.

**Comment by VenkatCisco** [comment]
> @VenkatCisco: Is it necessary to backport this to the 201911 branch? That branch is relatively stable; we only want to backport bug fixes.

updated to reflect only 202106 and 202012 branches. Our primary development branches are presently 202012 and master

**Comment by jleveque** [comment]
@VenkatCisco: Please fix conflict which arose from merging your other PRs.

**Comment by VenkatCisco** [comment]
> @VenkatCisco: Please fix conflict which arose from merging your other PRs.

Done. Thanks

**Review by jleveque — APPROVED** [review]

File: dockers/docker-platform-monitor/Dockerfile.j2 [hunk 1]
```diff
@@ -26,6 +26,7 @@ RUN apt-get update &&   \
         dmidecode       \
         i2c-tools       \
         psmisc          \
+        python-jsonschema \
```
Comment (lguohan) [inline dockers/docker-platform-monitor/Dockerfile.j2:29]: is this python2 version or python3 version? can you change to python3?

**Comment by anshuv-mfst** [comment]
@lguohan - could you please help with approval & merge. Build failures are fixed.

File: dockers/docker-platform-monitor/Dockerfile.j2 (hunk 1 above)
Comment (VenkatCisco) [reply dockers/docker-platform-monitor/Dockerfile.j2:29]: jsonschema is an implementation of JSON Schema for Python (supporting 2.7+ including Python 3). https://python-jsonschema.readthedocs.io/en/stable/

**Review by lguohan — APPROVED** [review]

**Pull request merged**